src/main/java/edu/ucsb/cs156/gauchoride/entities/DriverShift.java

```diff
@@ -0,0 +1,28 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "drivershifts")
+public class DriverShift {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long driverid;
```

Comment:
Why is one of this lower case long and the other capital Long?  Is that arbitrary, or is there a reason for the choice?